simulation-construction-set/falling-brick/src/main/java/us/ihmc/fallingBrick/FallingBrickRobot.java

```diff
@@ -0,0 +1,204 @@
+package us.ihmc.fallingBrick;
+
+import us.ihmc.euclid.tuple3D.Vector3D;
+import us.ihmc.graphicsDescription.Graphics3DObject;
+import us.ihmc.graphicsDescription.appearance.AppearanceDefinition;
+import us.ihmc.graphicsDescription.appearance.YoAppearance;
+import us.ihmc.jMonkeyEngineToolkit.GroundProfile3D;
+import us.ihmc.simulationconstructionset.FloatingJoint;
+import us.ihmc.simulationconstructionset.GroundContactModel;
+import us.ihmc.simulationconstructionset.GroundContactPoint;
+import us.ihmc.simulationconstructionset.Joint;
+import us.ihmc.simulationconstructionset.Link;
+import us.ihmc.simulationconstructionset.Robot;
+import us.ihmc.simulationconstructionset.util.LinearGroundContactModel;
+import us.ihmc.simulationconstructionset.util.RobotController;
+import us.ihmc.yoVariables.registry.YoVariableRegistry;
+import us.ihmc.yoVariables.variable.YoDouble;
+
+public class FallingBrickRobot extends Robot implements RobotController
+{
+   private static final double BASE_H = 0.1, BASE_W = 0.2, BASE_L = 0.3;
+   private static final double B1 = BASE_H / 2.0;
+   private static final double M1 = 1.7;
+   private static final double Ixx1 = 0.1, Iyy1 = 0.5, Izz1 = 0.9;
+   private static final double G = 9.81;
+
+   private final YoVariableRegistry registry = new YoVariableRegistry("FallingBrickController");
+
+   // position, velocity, and acceleration variables
+   YoDouble q_x, q_y, q_z, qd_x, qd_y, qd_z, qdd_x, qdd_y, qdd_z;
+   YoDouble q_qs, q_qx, q_qy, q_qz, qd_wx, qd_wy, qd_wz, qdd_wx, qdd_wy, qdd_wz;
+
+   YoDouble q_qlength, theta_x;
```

Comment:
I feel like all these variables are not used, so all fields declared lines 30, 31, and 33